classes/context/PKPSectionDAO.php

```diff
@@ -36,6 +54,17 @@ abstract public function newDataObject();
      */
     abstract public function getById($sectionId, $contextId = null);
 
+    /**
+     * Check if a section exists
+     */
+    public function exists(int $sectionId, ?int $contextId = null)
+    {
+        return DB::table($this->_getTableName())
+            ->where($this->_getIdColumnName(), $sectionId)
+            ->when($contextId, fn (Builder $q) => $q->where($this->_getContextIdColumnName(), $contextId))
```

Comment:
Use strict checks in `->when` conditions to avoid ambiguity between `0` and `null`. Preferred form:
```php
->when($contextId !== null, fn (Builder $q) => $q->where($this->_getContextIdColumnName(), $contextId))
```